Model/IO/Export/Nastran/BulkData/LongBulkData.cpp

```diff
@@ -0,0 +1,27 @@
+#include "LongBulkData.h"
+#include "../../../../Common/CommonMeshRepresentation.h"
+
+
+LongBulkData::LongBulkData()
```

Comment:
More empty constructors and destructors. Consider using YAGNI in your development.